### DISCUSSION TITLE
WARN  deprecated: .Site.Social was deprecated in Hugo v0.124.0 and will be removed in a future release. Use .Site.Params instead.

### DISCUSSION THREAD

**Discussion by BunnySakura** [adityatelange/hugo-PaperMod#1570] [Q&A]
这个警告似乎和`socialIcons`有关，我删除这个字段就会消失。
The warning seems to be related to `socialIcons`, which goes away when I delete the field.

实际上参考Hugo代码实现以及[Issue 12228](https://github.com/gohugoio/hugo/issues/12228)、[Pull 11570](https://github.com/gohugoio/hugo/pull/11570)：
Actually refer to the Hugo code implementation as well [Issue 12228](https://github.com/gohugoio/hugo/issues/12228)、[Pull 11570](https://github.com/gohugoio/hugo/pull/11570) :

> This is the first step in deprecating the social key in site configuration. Our internal exposure is limited to the opengraph and twitter_cards templates. Site authors will receive a warning, generated by the relevant internal template, if they are using:

所以如何消除这个警告，谢谢。
So how to get rid of this warning, thanks.

**Comment by VashJuan** [comment]
Your answer seems to be at https://github.com/gohugoio/hugo/pull/11570

OLD CODE:
```
[social]
facebook_admin= 'foo'
```
or

```
[social]
twitter = 'bar'
```
should be changed to this NEW CODE:

```
[params.social]
facebook_admin= 'foo'
```
and

```
[params.social]
twitter = 'bar'
```
